plugins/kafka/librdkafka

```diff
@@ -0,0 +1 @@
+Subproject commit 3e1babf4f26a7d12bbd272c1cdf4aa6a44000d4a
```

Comment:
The last stable branch for librdkafka ("0.7") does not include support for Kafka 0.8 and doesn't have the C++ interface that the plugin uses.  My suggestion is that we continue to use the master branch of the fork you made but do not update the fork.  That basically freezes it.  Once a stable librdkafka branch is released (ETA mid-December) we can test it and switch.